Makefile.common

```diff
@@ -139,14 +147,14 @@ $(CSHARE)/%: $(_DBUILD_DEPS)
 
 ALL_PLUGIN_SOURCE_FILES=$(shell find $(COQ_SRC_DIR)/plugins -name '*.ml' -or -name '*.mli' -or -name '*.mlg') $(ALL_ML_SOURCE_FILES)
 
-_build/default/plugins/%.cmxs: $(ALL_PLUGIN_SOURCE_FILES) $(_DBUILD_DEPS)
+_build/default/plugins/%_plugin.cmxs: $(ALL_PLUGIN_SOURCE_FILES) $(_DBUILD_DEPS)
 	$(SHOW)'DUNE      $@'
-	$(HIDE)$(_DBUILD) $@
+	$(HIDE)$(_DBUILD) _build/install/default/lib/coq-core/plugins/$(addsuffix _plugin.cmxs, $(shell basename $*)/$(shell basename $*))
```

Comment:
why not 
```suggestion
	$(HIDE)$(_DBUILD) _build/install/default/lib/coq-core/plugins/$(basename $*)/$(basename $*)_plugin.cmxs
```
?

In fact what was wrong with the old `$@` rule?

Reply:
I dispatch to `_build/install` since findlib need it to be there.